rfcs/20210827-benchdnn-model-support/README.md

```diff
@@ -0,0 +1,88 @@
+# Benchdnn as a new tool to measure full model performance.
+
+## Problem Statement.
+There are talks around and inside oneDNN team about the need to improve the way
+performance validation/benchmarking of E2E models or their parts can be done.
+
+Currently, benchdnn provides the only way to measure performance - single
+problem is executed on same memory until certain conditions are reached. This
+allows to know a problem performance in "vacuum" environment - unrealistic
+conditions for end users. It shows how efficient the kernel is, but final
+performance values may not be users will see in the end, which depends on many
+factors. The problem of this approach that the result should be somehow scaled
+down to be closer to the "reality" but it's hard to say how much and how to
+estimate this ratio. Certain performance problems can't be found with such
+methodology, like cores trash-trafficking due to different blocking strategies
+in adjacent primitives, or data eviction from caches due to, e.g., different
+sets of weights, while a single problem fits well.
+
+The next validation step is to have oneDNN integration into a framework and run
+models they support. It may include framework overheads and contain integration
+issues which makes performance analysis harder to estimate how good the results
```

Comment:
"which makes performance analysis harder to estimate how good the results are" --> 
"which makes performance analysis harder in terms of how good the results are"